donkeycar/templates/complete.py

```diff
@@ -880,7 +880,31 @@ def add_camera(V, cfg, camera_type):
               outputs=['cam/image_array', 'cam/depth_array',
                        'imu/acl_x', 'imu/acl_y', 'imu/acl_z',
                        'imu/gyr_x', 'imu/gyr_y', 'imu/gyr_z'],
+              threaded=False)
+        
+    elif cfg.CAMERA_TYPE == "OAKD":
```

Comment:
I don't thing the oakd part is in main yet; so should not be in this PR.  I'll go look at the oakd tomorrow, but it should not be mixed in this PR.